pkg/timelock/scheduler.go

```diff
@@ -112,13 +112,17 @@ func (tw *Worker) updateSchedulerDelay(t time.Duration) {
 
 // addToScheduler adds a new CallSchedule operation safely to the store.
 func (tw *Worker) addToScheduler(op *contract.TimelockCallScheduled) {
+	tw.mu.Lock()
```

Comment:
these changes to the `scheduler.go` module is due to a race condition uncovered by the new integration test